projects/melon-runtime/Melon.Engine/Builder/EngineBuilder.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Melon.Library.Static;
+
+namespace Melon.Engine.Builder
+{
+    public class EngineBuilder
+    {
+        private readonly Jint.Engine _engine;
+
+        public EngineBuilder()
+        {
+            var internalBinding = InternalBinding.Dictionary;
+
+            _engine = new();
+            _engine.SetValue("_$internalBinding", internalBinding);
+        }
+
+        public EngineBuilder Load(string identifier)
+        {
+            var content = LibraryLoader.ByIdentifier(identifier);
+            _engine.Execute(content);
+
+            return this;
+        }
+
+        public Jint.Engine Build()
+        {
+            return _engine;
+        }
```

Comment:
Minor detail: `.Build()` just returning the internal Jint instance is a bit weird.
I'd expect to have a method like `.AddLibrary(item)` to just append the libraries to the list, and `.Build()` just loads each item in the list.

Reply:
3d4a917a32ab9bb5d3e9af303c9101d379a520c6